package.json

```diff
@@ -1,10 +1,26 @@
 {
-  "name": "usine-distribuee",
-  "version": "0.1.0",
-  "repository": "git@github.com:CaenCamp/usine-distribuee.git",
-  "license": "GPL-3.0",
-  "private": true,
-  "workspaces": [
-    "apps/*"
-  ]
+	"name": "usine-distribuee",
+	"version": "0.1.0",
+	"repository": "git@github.com:CaenCamp/usine-distribuee.git",
+	"license": "GPL-3.0",
+	"private": true,
+	"workspaces": [
+		"apps/*"
+	],
+	"scripts": {
+		"lint": "prettier --check \"./**/*.js\"",
+		"lint:fix": "prettier --write \"./**/*.js\""
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "yarn lint"
```

Comment:
In most project, there use hysky with lint-staged (https://prettier.io/docs/en/precommit.html)

ex:
```
 "husky": {
        "hooks": {
            "pre-commit": "lint-staged"
        }
    },
    "lint-staged": {
        "*.js": "eslint",
        "*.{js,md,json}": "prettier --write"
    }
```